src/index.js

```diff
@@ -0,0 +1,41 @@
+(function($) {
+  var apiUrl = "https://tagchatter.herokuapp.com";
+
+  function fetchParrotsCount() {
+    return fetch(apiUrl + "/messages/parrots-count")
+      .then(function(response) {
+        return response.json();
+      })
+      .then(function(count) {
+        $("#parrots-counter").text(count);
+      });
+  }
+
+  function listMessages() {
+    // Faz um request para a API de listagem de mensagens
+    // Atualiza a o conteúdo da lista de mensagens
+    // Deve ser chamado a cada 3 segundos
```

Comment:
esse esquema de chamar a cada 3 segundos, not sure, mas acho que eles terão problemas com scroll se ficarem trocando as mensagens...Mas ai cabe a eles talvez pensar sobre isso e tentar uma solução de atualizar somente o necessário?

Reply:
humm, verdade. Eu voto pra deixar esse requisito e ver se alguem faz algo diferente